packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -199,6 +200,8 @@ class AndroidView extends StatefulWidget {
 ///
 /// Construction of UIViews is done asynchronously, before the UIView is ready this widget paints
 /// nothing while maintaining the same layout constraints.
+/// 
```

Comment:
looks like there's a whitespace at the end of this line that makes cirrus mad. :)

Reply:
Good catch! Thanks. VSCode always generates `/// ` with an empty space in the end when hit return key on the keyboard after a line of dart doc.